examples/ex22.cpp

```diff
@@ -0,0 +1,477 @@
+//                               MFEM Example 22
```

Comment:
Add companion PR in mfem/web with description and image for the new example.

Reply:
Done